### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ajaysubra**
# Description

Fixed the ordering of deep links (URL scheme and universal links) on the react native readme.

# Check List

- [ ] Are you changing anything with the public API?
- [ ] Are your changes backwards compatible with previous SDK Versions?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,16 +255,17 @@ instructions below.
...
 
   ```objective-c
   [RCTLinkingManager application:application openURL:url options:options]
   ```
 
-  Since you won't have `options`, you can just pass in an empty dictionary for that parameter.
-
-  With option 2, when you handle the open url (in [`application(_:open:options)`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623112-application)),
-  you call the linking code block above similar to what you would do with option 1.
+  For application, you can pass in an instance of `UIApplication` and since you won't have `options`, you can just pass in an empty dictionary for that parameter.
```
Comment (ndurell) [inline README.md:268]: should we just update the code too?

Comment (ajaysubra) [reply README.md:268]: Yup. doing that in the example app PR - https://github.com/klaviyo/klaviyo-react-native-sdk/pull/104

Comment (ajaysubra) [reply README.md:268]: Oh you mean the sample code in the readme?

Comment (ndurell) [reply README.md:268]: Yah just worried someone would copy/paste it wrong. But they could probably figure it out.

**Review by ndurell — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ajaysubra) [reply README.md:268]: added code, let me know if that's what you meant.

**Pull request merged**